<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GW2 Hub</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background:transparent;
}

QMenuBar
 {
    background-color: rgb(64, 64, 64);
}

QMenuBar::item 
{
    background-color: rgb(64, 64, 64);
	color: white;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background:  rgb(144, 144, 144);
}

QMenuBar::item:pressed {
    background:   rgb(144, 144, 144);
}

QMenu::item 
{
    background-color:  rgb(64, 64, 64);
	color: white;
}

QMenu::item:selected 
{
    background: rgba(100, 100, 100, 150);
}

QMenu::item:disabled
{
	background:  rgb(64, 64, 64);
   
	color: rgb(108, 108, 108);
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="FrameStyleSheet">
      <property name="styleSheet">
       <string notr="true">QFrame
{
	background-color: rgb(64, 64, 64);
    border: 1px solid black;
}

QPushButton
{
	background-color: rgb(136, 136, 136);
	color: white;
	border: 1px solid black;
	padding: 2px;
}


QPushButton:pressed 
{
    background-color: rgb(64, 64, 64);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="notesPushButton">
           <property name="text">
            <string>Notes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="materialTrackPushButton">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Material Tracker</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browserPushButton">
           <property name="text">
            <string>Browser</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose_Application"/>
   </widget>
   <widget class="QMenu" name="menuAPI">
    <property name="title">
     <string>API</string>
    </property>
    <addaction name="actionAdd_API_Key"/>
    <addaction name="actionRemove_API_Key"/>
   </widget>
   <widget class="QMenu" name="menuBrowser">
    <property name="title">
     <string>Browser</string>
    </property>
    <addaction name="actionDelete_Cookies"/>
    <addaction name="actionClear_All_Visited_Links"/>
    <addaction name="actionClear_History"/>
   </widget>
   <widget class="QMenu" name="menuMaterial_Tracker">
    <property name="title">
     <string>Material Tracker</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
    <addaction name="actionImport_Excel_Sheet"/>
   </widget>
   <widget class="QMenu" name="menuNotes">
    <property name="title">
     <string>Notes</string>
    </property>
    <addaction name="notesActionSave"/>
    <addaction name="notesActionOpen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAPI"/>
   <addaction name="menuNotes"/>
   <addaction name="menuMaterial_Tracker"/>
   <addaction name="menuBrowser"/>
  </widget>
  <action name="actionAdd_API_Key">
   <property name="text">
    <string>Add API Key</string>
   </property>
  </action>
  <action name="actionRemove_API_Key">
   <property name="text">
    <string>Remove API Key</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionClose_Application">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDelete_Cookies">
   <property name="text">
    <string>Delete Cookies</string>
   </property>
  </action>
  <action name="actionClear_All_Visited_Links">
   <property name="text">
    <string>Clear All Visited Links</string>
   </property>
  </action>
  <action name="actionClear_History">
   <property name="text">
    <string>Clear History</string>
   </property>
  </action>
  <action name="notesActionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="notesActionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionImport_Excel_Sheet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import Excel Sheet</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
